sysdig/resource_sysdig_secure_team.go

```diff
@@ -0,0 +1,167 @@
+package sysdig
+
+import (
+	"github.com/draios/terraform-provider-sysdig/sysdig/secure"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"strconv"
+	"time"
+)
+
+func resourceSysdigSecureTeam() *schema.Resource {
+	timeout := 30 * time.Second
+
+	return &schema.Resource{
+		Create: resourceSysdigTeamCreate,
+		Update: resourceSysdigTeamUpdate,
+		Read:   resourceSysdigTeamRead,
+		Delete: resourceSysdigTeamDelete,
+
+		Timeouts: &schema.ResourceTimeout{
+			Create: schema.DefaultTimeout(timeout),
+		},
+
+		Schema: map[string]*schema.Schema{
+			"theme": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Default:  "#73A1F7",
+			},
+			"name": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"description": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"scope_by": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Default:  "container",
+			},
+			"filter": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"use_sysdig_capture": {
+				Type:     schema.TypeBool,
+				Optional: true,
+				Default:  true,
+			},
+			"user_roles": {
+				Type:     schema.TypeSet,
+				Optional: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"email": {
+							Type:     schema.TypeString,
+							Required: true,
+						},
+
+						"role": {
+							Type:     schema.TypeString,
+							Optional: true,
+							Default:  "ROLE_TEAM_STANDARD",
```

Comment:
I set "ROLE_TEAM_STANDARD" as default role. is it ok?
https://docs.sysdig.com/en/user-and-team-administration.html#UUID-b7b6f0fb-85ec-5366-e3e4-69260be044fd_UUID-87f1d5be-77eb-4f1b-51d9-15612ba9d289

Reply:
Yes, it's correct :slightly_smiling_face: